### PR TITLE
[No-JIRA] Add tracking docs for android

### PR DESCRIPTION
**Pull request by amolgupta**
Adds tracking docs for Android

### CODE REVIEW

File: decisions/tracking.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Support for Tracking
+
+Type-ramp system has following values for tracking
```
Comment (k0nserv) [inline decisions/tracking.md]: How about "The type-ramp uses the following tracking values for tracking:"?

File: decisions/tracking.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+font size | letter Spacing
```
Comment (k0nserv) [inline decisions/tracking.md]: We should mention that this is the letter spacing as defined in Sketch. We could also calcualted the em tracking which is what Android uses and add it as a column. Tools like lillustrator/photoshop use thousands of em which can also be calculated.

File: decisions/tracking.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Support for Tracking
+
+The type-ramp system uses the following values for tracking:
```
Comment (shaundon) [inline decisions/tracking.md]: Two words 'type-ramp' please 😇

File: decisions/tracking.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+30	| -0.8|-0.02666
+36	| -1.0|-0.02778
+42	| -1.2|-0.02857
+52  | -1.4|-0.02692
+62  | -1.6|-0.02580
+74  | -1.8|-0.02432
+
+## Android
+
+For android the value for `letterSpacing` is calculated as
```
Comment (shaundon) [inline decisions/tracking.md]: Capitalised Android 😇

**Comment by shaundon** [comment]
Otherwise it's looking 💯

**Pull request merged**